Add dependency to libexecinfo and the associated version for backtraces.

diff -Nurp a/dmd/src/dmd/link.d b/dmd/src/dmd/link.d
--- a/dmd/compiler/src/dmd/link.d
+++ b/dmd/compiler/src/dmd/link.d
@@ -628,6 +628,8 @@
             argv.push("-lrt");
             // Link against libdl for phobos usage of dlopen
             argv.push("-ldl");
+            // Musl
+            argv.push("-lunwind");
         }
         else version (OpenBSD)
         {
